src/search/landmarks/landmark_factory_relaxation.cc

```diff
@@ -93,8 +93,7 @@ void LandmarkFactoryRelaxation::calc_achievers(
 
         vector<vector<int>> lvl_var;
         vector<utils::HashMap<FactPair, int>> lvl_op;
```

Comment:
Is `lvl_op` still needed? If not, I wonder why this doesn't generate a warning.

Reply:
No, it's not. I'm also surprised there was no warning about this... It makes me wonder whether there are other instances that we overlooked.